deploy/examples/create-external-cluster-resources.py

```diff
@@ -1750,6 +1752,27 @@ def gen_shell_out(self):
         shOutIO.close()
         return shOut
 
+    def gen_initial_data(self):
+        # TBD: add non-secured data keys
+        non_secured_data_keys = [
+            "ROOK_EXTERNAL_CEPH_MON_DATA",
+            "MONITORING_ENDPOINT",
+            "MONITORING_ENDPOINT_PORT",
+            "RADOS_NAMESPACE",
+            "RBD_POOL_NAME",
+            "RBD_METADATA_EC_POOL_NAME",
+            "TOPOLOGY_POOLS",
+            "TOPOLOGY_FAILURE_DOMAIN_LABEL",
+            "TOPOLOGY_FAILURE_DOMAIN_VALUES",
+            "RGW_ENDPOINT"
+        ]
+
+        initial_data = f"[InitialData]\n"
```

Comment:
is this needed?

Reply:
I kept the format same as we have for user provided configuration
https://github.com/rook/rook/blob/40832826877151a8745ab61375ad381ea7870aad/deploy/examples/create-external-cluster-resources.py#L1567